Lesson 1/code/exercises/exercise2.html

```diff
@@ -39,25 +39,42 @@
     <script type="text/javascript">
         
         var arr = [];
+        var cb=0;
+        var cn=0;
+        var cs=0;
+        var cf=0;
         
+	        
         function countBoolean() {
+            
+            if(typeof arr[arr.length-1] == "boolean")
+            {cb=cb+1;}

```

Comment:
There are some more elegant way to do it. You are really don't need to have counter variables. `array.forEach()` would be sufficient here.